### PR TITLE
Adds a Monki-Type Sechailer distributed through maint loot

### PR DESCRIPTION
**Pull request by terranaut1**
## About The Pull Request

adds a /sechailer/monkey and related sounds
adds it to the maint loot table

## Why It's Good For The Game

monki noise funny hehe

## Changelog
:cl:
add: Maintenance loot now has a chance to spawn one of the fabled Banana-Tors, as seen on TV during the Toolboxening!
/:cl:

### CODE REVIEW

**Comment by HiKewne** [comment]
me like monki

**Comment by PepperPrepper** [comment]
Cheers for using my monkey lines! I put some effort into them and got sad when the old pr got stale.

**Comment by terranaut1** [comment]
Thank you for providing them, I lack creativity. I also felt bad when I saw the old PR went stale with your suggestions unused

File: code/modules/clothing/masks/hailer.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // **** Security gas mask ****
 
 /obj/item/clothing/mask/gas/sechailer
-	name = "security gas mask"
+	name = "Security Compli-o-nator"
```
Comment (Dorsisdwarf) [inline code/modules/clothing/masks/hailer.dm]: Aw

Comment (terranaut1) [reply code/modules/clothing/masks/hailer.dm]: It's a result of me changing the output to use [src] so I can copy the proc, I can change name back if you want but the text phrase won't look as cool in chat then

Comment (Dorsisdwarf) [reply code/modules/clothing/masks/hailer.dm]: Your way makes more sense, actually

**Comment by kriskog** [comment]
Eww this again

**Comment by ninjanomnom** [comment]
This and the previous pr has shown I think that none of the maintainers want to deal with this or have it in game, find a maintainer who wants to support this and reopen the pr but as it is this isn't wanted.

**Pull request closed without merging**